Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/MainActivity.java

```diff
@@ -243,8 +247,22 @@ public void handleMessage(Message msg) {
                 startActivityForResult(intent, BluetoothState.REQUEST_CONNECT_DEVICE);
                 break;
             case R.id.menuUDPConnect:
+                ConnectivityManager cm = (ConnectivityManager) getSystemService( Context.CONNECTIVITY_SERVICE );
+                if ( Build.VERSION.SDK_INT >= Build.VERSION_CODES.M ) {
+                    for ( Network net : cm.getAllNetworks() ) {
+                        NetworkInfo networkInfo = cm.getNetworkInfo( net );
+                        if ( networkInfo.getType() == ConnectivityManager.TYPE_WIFI ) {
+                            cm.bindProcessToNetwork( net );
+                        }
+                        break;
```

Comment:
The break should be inside the if statement, otherwise only the first network is evaluated, which might be the mobile network.
With that change it works perfectly for me!